src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <span class="logo">
+        <a href=""><img src="images/logo.png" alt="logo"></a>
+      </span>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="" class="nav__link is-active">Apple</a>
```

Comment:
Put mock links for all of your links, e.g. #apple
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
```